src/commands/DensityProfile.hpp

```diff
@@ -39,7 +39,9 @@ class DensityProfile final: public AveCommand {
 private:
     Options options_;
     chemfiles::Selection selection_;
-    Axis axis_;
+    size_t n_axis_;
+    Axis axis_x_;
```

Comment:
Maybe call the axes `axis1_` and `axis2_`? They don't always refer to *x* and *y*.